### PR TITLE
Portifolio desktop version

### PR DESCRIPTION
**Pull request by LDouglasOT**
**Task Completed**

- Worked on the desktop version for the portfolio project
- Added the full functionality of the form
- Fixed any broken mobile design during the desktop version implementation

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+    background-color: transparent;
+    width: 100%;
+    padding: 0;
+  }
+
+  .intro-wrapper-ul-li {
+    margin: 5px 10px;
+  }
+
+  .works {
```
Comment (Nyame-Wolf) [inline style.css:116]: Kindly note to meet this project milestone requirements, i suggest you use `display:grid` for your card items.
The grid should style cards both in mobile version and desktop versions.
Screenshot 2022-09-30 at 04 41 47

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @LDouglasOT  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,846 @@
...
+  }
+
+  .w-header span {
+    margin: 0 3px;
+  }
+
+  .works-ul {
+    list-style: none;
+    display: grid;
+    grid-template-columns: auto auto auto;
```
Comment (nedjwab) [inline style.css:527]: Nice use of the grid 🔥🔥🔥

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @LDouglasOT    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors.
- ✔good design 🔥🔥
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## [OPTIONAL CHANGES ] 📝

[ ] kindly adjust the `margin` in `my recent work section` to look cleaner.

margin

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**